package.json

```diff
@@ -62,6 +62,7 @@
     "enzyme-adapter-react-16": "^1.1.1",
     "eslint": "^4.15.0",
     "eslint-plugin-react": "^7.5.1",
+    "humanize-plus": "^1.8.2",
```

Comment:
I don't see this being used anywhere in this PR, why include it?

Reply:
its in the next PR - i wanted to clear as many commits as possible - and use of this lib seemed like a separate conv - so i pushed it up front

Reply:
I'd rather keep things in context, as I said in my review comment. Right now you have me confused because you add a dependency without any explanation of what that dep will be used for. I would prefer reading more code, but understand what the code is doing, than the opposite.

Reply:
ill shift this commit to where its required